Doppler.BigQueryMicroservice/Controllers/WeatherForecastController.cs

```diff
@@ -1,30 +0,0 @@
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
-using Doppler.BigQueryMicroservice.Weather;
-using Microsoft.AspNetCore.Mvc;
-using Microsoft.Extensions.Logging;
-
-namespace Doppler.BigQueryMicroservice.Controllers
-{
-    [ApiController]
-    [Route("[controller]")]
-    public class WeatherForecastController : ControllerBase
-    {
-        private readonly ILogger<WeatherForecastController> _logger;
-        private readonly WeatherForecastService _service;
-
-        public WeatherForecastController(ILogger<WeatherForecastController> logger, WeatherForecastService service)
-        {
-            _logger = logger;
-            _service = service;
-        }
-
-        [HttpGet]
-        public IEnumerable<WeatherForecast> Get()
```

Comment:
If you want, now you can remove this endpoint from your [demo.http](https://github.com/FromDoppler/big-query-microservice/blob/main/demo.http) file:
![image](https://user-images.githubusercontent.com/75738115/128757528-c9afe15d-3345-421b-b6ff-004457cea0e8.png)

Reply:
If accept my last suggestion you can remove `hello` endpoints too